webpack/assets/javascripts/react_app/components/notifications/NotificationDrawerToggle.js

```diff
@@ -23,9 +23,9 @@ class NotificationDrawerToggle extends Component {
 
   onChange(actionType) {
     switch (actionType) {
-      case ACTIONS.RECEIVED_NOTIFICATIONS: {
+      case ACTIONS.RECEIVED_NOTIFICATIONS:
+      case ACTIONS.MARK_AS_READ: {
         const notifications = NotificationsStore.getNotifications();
```

Comment:
fall through switch statements should not be allowed. If a single action name can't describe both cases, refactor and extract a method which can be called from each case.